pkg/generators/markers_test.go

```diff
@@ -369,16 +392,20 @@ func TestParseCommentTags(t *testing.T) {
 				`+k8s:validation:cel[1]:message="must be greater than 5"`,
 				`+k8s:validation:cel[1]:optionalOldSelf`,
 			},
-			expected: generators.CommentTags{
-				CEL: []generators.CELTag{
-					{
-						Rule:    "oldSelf == self",
-						Message: "cant change",
-					},
-					{
-						Rule:            "self > 5",
-						Message:         "must be greater than 5",
-						OptionalOldSelf: ptr.To(true),
+			expected: &spec.Schema{
+				VendorExtensible: spec.VendorExtensible{
+					Extensions: map[string]interface{}{
+						"x-kubernetes-validations": []interface{}{
+							map[string]interface{}{
+								"rule":    "oldSelf == self",
+								"message": "cant change",
+							},
+							map[string]interface{}{
+								"rule":            "self > 5",
+								"message":         "must be greater than 5",
+								"optionalOldSelf": true,
```

Comment:
We're not losing any information now that this isn't a pointer right?

Reply:
correct. since the extensions map is intended to represent the JSON structure of things, I also think it is better that it is not a pointer